blob/store.go

```diff
@@ -31,12 +32,19 @@ import (
 // Implementations of this interface must be safe for concurrent use by
 // multiple goroutines.
```

Comment:
May want some specific interface docs either top-level or on the methods that KV and CAS names are a single namespace, i.e. you can access the same keyset through both method sets?

Reply:
Oh yes, thanks for the reminder, I meant to add this.